src/borg/archiver/check_cmd.py

```diff
@@ -182,9 +180,11 @@ def build_parser_check(self, subparsers, common_parser, mid_common_parser):
 
         If ``--repair --undelete-archives`` is given, Borg will scan the repository
```

Comment:
guess this should be renamed to --find-lost-archives.

if users delete archives, they can easily `borg undelete` these.
check does something different: with this option it finds archives for which we have lost the entry in archives/ directory.

Reply:
fixed